webui/react/src/services/api.ts

```diff
@@ -2,8 +2,9 @@
 import * as DetSwagger from '@determined-ai/api-ts-sdk';
 import { CancelToken } from 'axios';
 
-import { generateApi } from 'services/apiBuilder';
+import * as DetSwagger2 from 'services/api-ts-sdk';
 import { processApiError } from 'services/apiBuilder';
+import { generateApi } from 'services/apiBuilder';
```

Comment:
chore: can collapse into one line

Reply:
nit: can't remember (you mentioned something about this line being temporary), but any reason why this isn't:
```
import { generateApi, processApiError } from 'services/apiBuilder';
```